supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -0,0 +1,26 @@
+[[support]]
+= Support
+
+[discrete]
+== Technology Preview
+
+Technology Preview documentation guidelines apply to any form of early access feature that is not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.
```

Comment:
Is this first sentence a true statement? I know that we have other types of non-supported levels, like dev preview. So I don't think we necessarily want people to label things TP if they're actually some other level. 

I'd recommend removing this first sentence and starting with a definition of what TP is, then leading into the guidelines. Maybe something like:

```suggestion
Technology Preview features provide early access to upcoming product innovations, enabling customers to test functionality and provide feedback during the development process. However, these features are not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.

When documenting a Technology Preview feature, follow these guidelines:
```